supporter-product-data/cloudformation/cfn.yaml

```diff
@@ -267,7 +267,7 @@ Resources:
                    {
                     "ErrorEquals": ["States.ALL"],
                     "IntervalSeconds": 60,
-                    "MaxAttempts": 20,
+                    "MaxAttempts": 40,
```

Comment:
Could we maybe leave a comment for future folk that this is something we expect in the unusual (although regular) circumstances of the morning hours?

Reply:
I'm not sure it's possible here as it's json! An incentive to migrate to cdk...